x-pack/plugins/cases/public/components/visualizations/types.ts

```diff
@@ -7,4 +7,12 @@
 
 import type { TypedLensByValueInput } from '@kbn/lens-plugin/public';
 
-export type LensProps = Pick<TypedLensByValueInput, 'attributes' | 'timeRange'>;
+export type LensProps = Pick<TypedLensByValueInput, 'attributes' | 'timeRange'> & {
+  /**
+   * Optional metadata used to customize the Lens Attachment rendering.
+   */
+  metadata?: {
+    description?: string;
+    width?: string | number;
```

Comment:
The `width` seems like a UI property unrelated to the visualization. It seems to belong to a `prop` rather than a data property. From an API perspective, the `width` users should not be concerned about what value they should put there. What do you think about not persisting the `width` here and leave the UIs component to full width until we figure out a way to configure it?

Reply:
I don't understand how the visualisation width is unrelated to the visualisation. I already added the `width` to the store state because Design requested that we keep similar proportions when adding visualisations to cases:
Like here:
![Screenshot 2024-01-29 at 15 10 37](https://github.com/elastic/kibana/assets/1490444/528a876b-280c-41a1-9cd5-b74a1929744a)

But that isn't a priority. So, I will delete the `width` property to unblock this PR.

Reply:
That is how the visualization looks on a wide screen:
![Screenshot 2024-01-29 at 15 25 11](https://github.com/elastic/kibana/assets/1490444/498a07fb-cd1b-4e21-9282-b6c99c925a29)

Reply:
> I don't understand how the visualization width is unrelated to the visualization
I mean that it is unrelated to the data of the visualization from an API perspective. Adding the `width` to the data raises a lot of questions like "Width for what?", "For which screen?", "For which component?", "If a user fetches this data from our APIs and tries to render the visualization with his/her own custom tools what does this width mean for their user case?", etc. The design makes sense. I think is better to find another way (on another PR) to do it on the UI level without the need to persist the value.